python/xorbits/_mars/dataframe/initializer.py

```diff
@@ -41,6 +41,28 @@
 cudf = lazy_import("cudf")
 
 
+def _check_expr_in_list_of_list(x: list):
+    for i in x:
+        if isinstance(i, list):
+            for item in i:
+                if (
+                    hasattr(item, "data")
+                    and isinstance(item.data, TensorData)
+                    and item.data.ndim == 0
+                ):
+                    return True
+    return False
+
+
+def _convert_data_to_dict(data, cols):
```

Comment:
```suggestion
def _convert_data_to_dict(data: List[list], cols: list) -> DIct[Any, list]:
```

Type annotation should be added.